components/gfx/Cargo.toml

```diff
@@ -28,6 +28,7 @@ msg = {path = "../msg"}
 net_traits = {path = "../net_traits"}
 ordered-float = "0.4"
 range = {path = "../range"}
+script = {path = "../script"}
```

Comment:
What's a way to go about this? I also didn't like writing this line but I don't know how else I would solve the problem. Maybe `gfx` should just not make any requests?